### PR TITLE
Switch UR10, UR10e and UR16e to effort controllers

### PR DESCRIPTION
**Pull request by ipa-kut**
This fix addresses https://github.com/ros-industrial/universal_robot/issues/521 which in turn addresses https://github.com/ros-industrial/universal_robot/issues/386, and was worked on during WRID2020

Based on a rough understanding of [this comment](https://github.com/ros-industrial/universal_robot/pull/504#issuecomment-652394378), only the three URs mentioned in the title have been updated.

PID values for the effort controllers were taken from [here](https://github.com/fmauch/universal_robot/commit/0d2ec11e2ac4a55317ffe427e16c1a949a1a4bee) 

After making these changes, the gazebo simulation of the robot was observed to start in a "weird" pose :
UR16e:   
initial_pose_16e

UR10 and UR10e:   
initial_pos_10e

However, the controllers initially seem to initially work well, and the robots could be jogged using the [RQT Joint trajectory controller plugin](http://wiki.ros.org/rqt_joint_trajectory_controller)

### CODE REVIEW

**Comment by fmauch** [comment]
As this PR is branching of `melodic-devel-staging` none of the models use the updated cog from #504.

Eventually, this should be changed for all 7 robot variants, though.

**Comment by ipa-kut** [comment]
I will change this for the remaining URs now.

> As this PR is branching of `melodic-devel-staging` none of the models use the updated cog from #504.

Would it be better to wait until #504 is merged, to rebase this and test again?

**Comment by fmauch** [comment]
> I will change this for the remaining URs now.
> 
> > As this PR is branching of `melodic-devel-staging` none of the models use the updated cog from #504.
> 
> Would it be better to wait until #504 is merged, to rebase this and test again?

I think, this can be implemented independently from #504 but for fine tuning it might be interesting whether it makes a difference whether we use the cylinder approximation for the inertia or we use the matrices provided by the robot controller. So, testing and fine-tuning this might require a rebase on #504. However, as the results from those tests will also impact #504, we should not wait for that to be merged, but develop them in parallel. I think, this won't be too much of an issue as there are no conflicts between those two.

**Review by ipa-nhg — APPROVED** [review]
Tested with https://github.com/ros-industrial/universal_robot/pull/526 and it works.

Great contribution @ipa-kut, thanks! :+1:

**Comment by RobertWilbrandt** [comment]
The changes from this should now be merged through #619. Thanks @ipa-kut for getting this started!

**Pull request closed without merging**
